contracts/upgradeable_contracts/omnibridge_nft/components/native/NFTMediatorBalanceStorage.sol

```diff
@@ -8,27 +8,38 @@ import "../../../../upgradeability/EternalStorage.sol";
  */
 contract NFTMediatorBalanceStorage is EternalStorage {
     /**
-     * @dev Tells if mediator owns the given token. More strict than regular token.ownerOf() check,
+     * @dev Tells amount of owned tokens recorded at this mediator. More strict than regular token.ownerOf()/token.balanceOf() checks,
      * since does not take into account forced tokens.
      * @param _token address of token contract.
      * @param _tokenId id of the new owned token.
-     * @return true, if given token is already accounted in the mediator balance.
+     * @return amount of owned tokens, 0 or 1 for regular NFTs.
```

Comment:
Is it worth to use "ERC721 NFTs" instead of "regular NFTs"?

Reply:
Sure.